Sources/SentryCrash/Recording/SentryCrash.m

```diff
@@ -67,10 +67,19 @@
 static NSString *
 getBundleName(void)
 {
-    NSString *bundleName = [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleName"];
+    static NSString *bundleName = nil;
+
     if (bundleName == nil) {
-        bundleName = @"Unknown";
+#if TEST
+        bundleName = @"Sentry/Test";
+#else
+        bundleName =
+            [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleName"] ?: @"Unknown";
+#endif
```

Comment:
`m`: I would prefer finding a different way of testing this. One way would be to define an extra method or class that handles the edge case below and directly call it in tests.